apollo-router-core/src/traits.rs

```diff
@@ -70,11 +70,11 @@ where
 
 impl<T: ?Sized> WithCaching for T where T: QueryPlanner + Sized + 'static {}
 
-/// An object that accepts a [`Request`] and allow creating [`PreparedQuery`]'s.
+/// An object that accepts a [`Request`] and allows creating [`PreparedQuery`]'s.
```

Comment:
NIT: Slightly better grammar to use "allows for".

Reply:
```suggestion
/// An object that accepts a [`Request`] and allows for creating [`PreparedQuery`]'s.
```